tchannel/tornado/connection.py

```diff
@@ -555,13 +555,12 @@ def stream_request(self, request):
         """send the given request and response is not required"""
         request.close_argstreams()
 
-        stream_future = self._stream(request, self.request_message_factory)
-
-        IOLoop.current().add_future(
-            stream_future,
-            lambda f: request.close_argstreams(force=True)
-        )
+        def on_done(future):
+            future.exception()  # < to avoid "unconsumed exception" logs
```

Comment:
Yeah, same thing we were doing before. I'm reluctant to add new possibly noisy log entries in this part of the system.